docs/Configuration.rst

```diff
@@ -59,3 +59,13 @@ Configuration
     **Default:** ``nil``
 
     If true, format outputted queries with newlines and colors to be more easily readable by humans
+
+  **record_timestamps**
+    **Default:** ``false``
+    
+    A Rails-inspired configuration to manage inclusion of the Timestamps module. If set to true, all ActiveNode and ActiveRel models will include the Timestamps module and have ``:created_at`` and ``:updated_at`` properties. For example, include the following in your config: ``Neo4j::Config.record_timestamps = true``.
```

Comment:
Actually, I just realized that my suggestion is Rails-specific.  I'll put in some docs at the top which make it clear.  Thanks!